recipes/jasper/all/conanfile.py

```diff
@@ -77,11 +77,11 @@ def generate(self):
         if Version(self.version) >= "3.0.0":
             tc.variables["JAS_ENABLE_LIBHEIF"] = False
         tc.variables["JAS_ENABLE_OPENGL"] = False
-
         if cross_building(self):
             tc.cache_variables["JAS_CROSSCOMPILING"] = True
             tc.cache_variables["JAS_STDC_VERSION"] = "199901L"
-
+        if Version(self.version) >= "4.2.0":
+            tc.variables["JAS_PACKAGING"] = True
```

Comment:
Some of the credits definitely go to @uilianries 👏 

https://github.com/jasper-software/jasper/pull/347

Reply:
Open source community rocks! 🚀